tests/components/Draft.test.jsx

```diff
@@ -6,7 +6,7 @@ let mountedComponent;
 
 props = {
   draftOrders: [
-    { uuid: 6, display: 'hermatocrite' }
+    { uuid: 6, display: 'hermatocrite', set: 'panel', type: 'laborder' }
```

Comment:
To ensure that people that would be reading your test suits in the nearest future won't be confused, the `set` property of a lab order is actually a `boolean` ie `true` or `false`. Based on the behavior you are trying to achieve with this:
```
draftOrders: [
 { uuid: 6, display: 'hermatocrite', set: 'panel', type: 'laborder' }
]
```
You should change the `set` property to either `true` or `false`.

Reply:
Yes, thanks I wasn't sure of the data type for the `set` property.